src/styles/StyleUtils.ts

```diff
@@ -1142,11 +963,8 @@ function getEmojiReactionCounterTextStyle(hasUserReacted) {
 
 /**
  * Returns a style object with a rotation transformation applied based on the provided direction prop.
- *
- * @param {string} direction - The direction of the rotation (CONST.DIRECTION.LEFT or CONST.DIRECTION.RIGHT).
```

Comment:
You should leave the JSDoc if the parameter has some description, just remove the content with `{...}`

Reply:
Done